firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -288,7 +288,9 @@ class SearchTests: FeatureFlaggedTestBase {
             mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton])
             XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton].label, "New Tab")
             app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton].waitAndTap()
+            mozWaitForElementToNotExist(app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton])
```

Comment:
On iOS 15, the "cancel" button may be tapped too early intermittently. Let's confirm the new tab button is gone first.